### DISCUSSION TITLE
MathML

### DISCUSSION THREAD

**Discussion by diligentlystereotypedinvestment** [spacecowboy/Feeder#282] [Ideas]
I have a feature request: support MathML when reading articles using the built in reader.

**Comment by diligentlystereotypedinvestment** [comment]
Also, if someone could point me to where the built in reader implementation is, I could try my hand at adding it myself when I've got some time. I've seen a little bit of Kotlin Android development before.

**Reply by spacecowboy** [reply]
You should look at the table-layout branch where I'm rewriting the renderer. You'll see most commits touching it.

**Reply by spacecowboy** [reply]
I suspet your best bet would be adapting something that converts mathml to a vector image
